job_spec/VOLCSARVATORY_MULTI_BURST.yml

```
@@ -0,0 +1,78 @@
+VOLCSARVATORY_MULTI_BURST:
+  required_parameters:
+    - reference
+    - secondary
+  parameters:
+    reference:
+      api_schema:
+        type: array
```

```suggestion
        type: array
        minItems: 1
        maxItems: 30
```
set_batch_overrides will fail if there are more than 30 items in this array, and that will happen when the job is executed, so we should add this so the API reject those kinds of jobs

https://github.com/ASFHyP3/hyp3/blob/develop/apps/set-batch-overrides/src/set_batch_overrides.py#L47-L73